test/testing-utils.ts

```diff
@@ -1,7 +1,8 @@
+import cp from 'child_process'
```

Comment:
maybe `{ fork }`, to distinguish from `copy` 😂 (that's what I was thinking when i saw this